app/src/main/java/com/example/cutepets/MyApp.java

```diff
@@ -30,6 +30,7 @@
      * set android:name="&lt;this.class.name&gt;" attribute on your application element in
      * AndroidManifest.xml or you are not setting this.analytics field in onCreate method override.
      */
+    @SuppressWarnings("unused")
```

Comment:
If applicable, can you add a note in description about why warning is being suppressed.